Sources/MapboxMaps/Foundation/Extensions/CGSize.swift

```diff
@@ -0,0 +1,7 @@
+import Foundation
+
+extension CGSize {
+    internal var mbmSize: Size {
```

Comment:
why is this needed? Feels like a weird roundabout of conversions

Reply:
I agree, but it does make optional conversions easier. So from `CGSize?` to `Size?` is just `cgSize?.mbmSize`.

Reply:
I feel like this extension is not needed.